### PR TITLE
Indirect simultaneous docs

### LINKED ISSUE #27840: Update docs for simultaneous fitting in QENS
We need to update the user docs for simultaneous fitting in QENS interface

### PR DESCRIPTION
**Pull request by Matthew-Andrew**
**Description of work.**

The documentation for Indirect data analysis has been updated to match the new functionality introduced into the interface. Most notably Simultaneous fitting.

**To test:**
  * Build the docs and look at the Indirect Data Analysis documentation in the interfaces section. Check things make sense and that all the links work.

Fixes #27840  

*This does not require release notes* because documentation changes

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/interfaces/Indirect Fitting.rst [hunk 1]
```diff
@@ -0,0 +1,493 @@
...
+
+Next to the 'Plot' label, you can select a parameter to plot and then click 'Plot' to plot it with error 
+bars across the fit spectra (if multiple data-sets have been used, a separate plot will be produced for each data-set). 
+The 'Plot Output' options will be disabled after a fit if there is only one data point for the parameters.
+
+During a sequential fit, the parameters calculated for one spectrum become the start parameters for the next spectrum to be fitted. 
+Although this normally yields better parameter values for the later spectra, it can also lead to poorly fitted parameters if the
+next spectrum is not 'related' to the previous spectrum. It may be useful to replace this poorly fitted spectrum with the results 
+from a single fit using the 'Edit Result' option. 
+Clicking the 'Edit Result' button will allow you to modify the data within your *_Results* workspace using results
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
Clicking the 'Edit Result' button will allow you to modify the data within your *_Results* workspace using the results 
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+Next to the 'Plot' label, you can select a parameter to plot and then click 'Plot' to plot it with error 
+bars across the fit spectra (if multiple data-sets have been used, a separate plot will be produced for each data-set). 
+The 'Plot Output' options will be disabled after a fit if there is only one data point for the parameters.
+
+During a sequential fit, the parameters calculated for one spectrum become the start parameters for the next spectrum to be fitted. 
+Although this normally yields better parameter values for the later spectra, it can also lead to poorly fitted parameters if the
+next spectrum is not 'related' to the previous spectrum. It may be useful to replace this poorly fitted spectrum with the results 
+from a single fit using the 'Edit Result' option. 
+Clicking the 'Edit Result' button will allow you to modify the data within your *_Results* workspace using results 
+produced from a singly fit spectrum. See the algorithm :ref:`IndirectReplaceFitResult <algm-IndirectReplaceFitResult>`.
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
produced from a single fit. See the algorithm :ref:`IndirectReplaceFitResult <algm-IndirectReplaceFitResult>`.
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+
+Clicking the 'Save Result' button will save the result of the fit to your default save location.
+
+MSD Fit
+-------
+
+Given either a saved NeXus file or workspace generated using the Elwin tab, this
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
Given either a saved NeXus file, or workspace generated using the Elwin tab, this
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+tab fits :math:`intensity` vs. :math:`Q` with one of three functions for each
+run specified to give the Mean Square Displacement (MSD). It then plots the MSD
+as function of run number. This is done by means of the
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
as function of run number. This is done using the
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 5]
```diff
@@ -0,0 +1,493 @@
...
+
+MSDFit searches for the log files named <runnumber>_sample.txt in your chosen
+raw file directory (the name ‘sample’ is for OSIRIS). These log files will exist 
+if the correct temperature was loaded using SE-log-name in the Elwin tab. If they 
+exist the temperature is read and the MSD is plotted versus temperature; if they do 
+not exist the MSD is plotted versus run number (last 3 digits).
+
+The fitted parameters for all runs are in _msd_Table and the <u2> in _msd. To
+run the Sequential fit a workspace named <inst><first-run>_to_<last-run>_eq is
+created of :math:`intensity` v. :math:`Q` for all runs. A contour or 3D plot of
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
created, consisting of :math:`intensity` v. :math:`Q` for all the runs. A contour or 3D plot of
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 6]
```diff
@@ -0,0 +1,493 @@
...
+this may be of interest.
+
+A sequential fit is run by clicking the Run button at the bottom of the tab, a
+single fit can be done using the Fit Single Spectrum button underneath the
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
single fit can be performed using the Fit Single Spectrum button underneath the
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 7]
```diff
@@ -0,0 +1,493 @@
...
+preview plot. A simultaneous fit may be done in a very similar fashion by changeing the Fit Type to Simultaneous
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
preview plot. A simultaneous fit may be performed in a similar fashion by changing the Fit Type to Simultaneous 
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 8]
```diff
@@ -0,0 +1,493 @@
...
+
+2. Change the **Plot Spectrum** spinbox seen underneath the mini-plots to change the spectrum displayed
+   in the upper mini-plot.
+
+3. Change the **EndX** variable to be around 0.8 in order to change the Q range over which the fit shall
+   take place. Alternatively, drag the **EndX** blue line seen on the mini-plot using the cursor.
+
+4. Choose the **Fit Type** to be Gaussian. The parameters for this function can be seen if you
+   expand the row labelled **f0-MsdGauss**. Choose appropriate starting values for these parameters.
+   When you begin to edit parameters in addition to being able to change the value of the parameter there are two additional options.
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
   As well as being able to change the value of the parameters, two additional options are available.
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 9]
```diff
@@ -0,0 +1,493 @@
...
+   Clicking on the button with `...` will bring up more options to set constraints and ties on the parameters. The checkbox will toggle 
+   whether the parameter is local or global.
+
+5. Tick **Plot Guess** to get a prediction of what your fit will look like.
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
5. Tick **Plot Guess** to get a prediction of what the fit will look like.
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 10]
```diff
@@ -0,0 +1,493 @@
...
+
+6. Click **Run** and wait for the interface to finish processing. This should generate a
+   _Parameters table workspace and two group workspaces with end suffixes _Results and
+   _Workspaces. The mini-plots should also update, with the upper plot displaying the
+   calculated fit and the lower mini-plot displaying the difference between the input data and the
+   fit.
+
+7. Alternatively, you can click **Fit Single Spectrum** to perform a fit only for the spectrum
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
7. Alternatively, you can click **Fit Single Spectrum** to perform a fit for the spectrum
```

File: docs/source/interfaces/Indirect Fitting.rst [hunk 11]
```diff
@@ -0,0 +1,493 @@
...
+
+In both I(Q, t) Fit and ConvFit the following options are available when fitting
+using FABADA:
+
+Output Chain
+  Select to enable output of the FABADA chain when using FABADA as the fitting
+  minimizer.
+
+Chain Length
+  Number of further steps carried out by fitting algorithm once parameters have
```
Comment (StephenSmith25) [inline docs/source/interfaces/Indirect Fitting.rst]:
```suggestion
  Number of further steps carried out by fitting algorithm once the parameters have
```

**Review by StephenSmith25 — COMMENTED** [review]
The documentation pages look fine, just a few minor comments.

File: docs/source/interfaces/Indirect Analysis.rst [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+Calculate Errors
+  The calculation of errors using a Monte Carlo implementation can be skipped by unchecking
+  this option.
+
+Number Of Iterations
+  The number of iterations to perform in the Monte Carlo routine for error calculation 
+  in I(Q,t). 
+
+Run
+  Runs the processing configured on the current tab.
```
[the remark this answers is not in the document]
Comment (Mantid-Matthew) [reply docs/source/interfaces/Indirect Analysis.rst]: Confusingly this tab does not do a Fit. It does a fourier transform of the data instead so I think processing is more appropriate here.

File: docs/source/interfaces/Indirect Analysis.rst [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+  The workspace currently active in the preview plot.
+
+Spectrum
+  Changes the spectrum displayed in the preview plot.
+
+Plot Current Preview
+  Plots the currently selected preview plot in a separate external window
+
+Run
+  Runs the processing configured on the current tab.
```
[the remark this answers is not in the document]
Comment (Mantid-Matthew) [reply docs/source/interfaces/Indirect Analysis.rst]: Confusingly this tab does not do a Fit. It does a fourier transform of the data instead so I think processing is more appropriate here.

**Review by StephenSmith25 — APPROVED** [review]
Looks good now 👍

**Pull request merged**